### DISCUSSION TITLE
rustdesk Blocked in Iran ??!!

### DISCUSSION THREAD

**Discussion by MrDataCoder** [rustdesk/rustdesk#8068] [General]
Hello.  I hope you doing well.
Sadly! I see some problems to connect in directly from Iran IPs , I just wanted to be sure is this block network communications from yours or other reasons?
Thanks.

**Comment by rustdesk** [comment]
we do nothing, but i guess you are block by the cloud vendor, i guess you know the reason, no need to explain :(

**Reply by MrDataCoder** [reply]
OK, thanks.
I did tested this on at least 4 differents IPs and ISPs from Iran. 
I will investigate why I can't connect directly to rustdesk server.

**Reply by sepsol** [reply]
@MrDataCoder Were you finally able to resolve your issue?

**Reply by MrDataCoder** [reply]
@sepsol 
Hi
No I couldn't , right now using it with proxy.
